### PR TITLE
Fix some metric on dashboard

### PR DESCRIPTION
**Pull request by jkotalik**
And other small nits

### CODE REVIEW

File: samples/frontend-backend/frontend/Program.cs [hunk 1]
```diff
@@ -14,9 +14,6 @@ public static void Main(string[] args)
         {
             Activity.DefaultIdFormat = ActivityIdFormat.W3C;
             CreateHostBuilder(args).Build().Run();
-
```
Comment (davidfowl) [inline samples/frontend-backend/frontend/Program.cs:17]: This is the best part of this change wtf.

File: src/Microsoft.Tye.Hosting/ProcessRunner.cs [hunk 2]
```diff
@@ -397,6 +397,15 @@ async Task RunApplicationAsync(IEnumerable<(int ExternalPort, int Port, string?
                     {
                         _logger.LogError(0, ex, "Failed to launch process for service {ServiceName}", replica);
 
+                        if (!_options.Watch)
```
Comment (davidfowl) [inline src/Microsoft.Tye.Hosting/ProcessRunner.cs:400]: That doesn't seem like a nit

File: samples/frontend-backend/frontend/Program.cs (hunk 1 above)
Comment (rynowak) [reply samples/frontend-backend/frontend/Program.cs:17]: @JamesNK

File: src/Microsoft.Tye.Hosting/ProcessRunner.cs (hunk 2 above)
Comment (jkotalik) [reply src/Microsoft.Tye.Hosting/ProcessRunner.cs:400]: Yeah, should have also mentioned this.

File: samples/frontend-backend/frontend/Program.cs (hunk 1 above)
Comment (jkotalik) [reply samples/frontend-backend/frontend/Program.cs:17]: uhhh dotnet watch testing stuff.......

Comment (JamesNK) [reply samples/frontend-backend/frontend/Program.cs:17]: Stop right there! You are under arrest by the whitespace police

image

**Pull request merged**